app/server/models/organisation.py

```diff
@@ -51,18 +54,33 @@ class Organisation(ModelBase):
 
     custom_welcome_message_key = db.Column(db.String)
 
-
-    def __init__(self, **kwargs):
-        super(Organisation, self).__init__(**kwargs)
-
-        self.system_blockchain_address = bt.create_blockchain_wallet(
-            wei_target_balance=current_app.config['SYSTEM_WALLET_TARGET_BALANCE'],
-            wei_topup_threshold=current_app.config['SYSTEM_WALLET_TOPUP_THRESHOLD'],
-        )
-
     def send_welcome_sms(self, to_user):
         if self.custom_welcome_message_key:
             message = i18n_for(to_user, "organisation.{}".format(self.custom_welcome_message_key))
         else:
             message = i18n_for(to_user, "organisation.generic_welcome_message")
         message_processor.send_message(to_user.phone, message)
+
+
+    def __init__(self, is_master=False, **kwargs):
+
+        super(Organisation, self).__init__(**kwargs)
+
+        if is_master:
```

Comment:
Would there ever be more than one master organization?

Reply:
If not, then should probably be a check for existing master orgs first `master_organisation = Organisation.query.filter_by(is_master=True).first()`...

Reply:
Or just make is_master unique?

Reply:
good point. we may in theory want a way to update the master organisation, but for now i'll just throw an exception if there's already one.